package.json

```diff
@@ -17,6 +17,7 @@
     "can-stache-bindings": "^3.0.5",
     "can-types": "^1.0.0",
     "can-util": "^3.7.0",
+    "can-validate-interface": "^0.1.0",
```

Comment:
I agree with Justin. 

I've updated this to `0.x` so that it will get minor version updates and we'll change it to `^1.0.0` once it's publicly documented.